pkg/reconciler/configuration/configuration.go

```diff
@@ -203,7 +200,7 @@ func (c *Reconciler) reconcile(ctx context.Context, config *v1alpha1.Configurati
 		return err
 	}
 
-	return c.gcRevisions(ctx, config)
+	return nil
```

Comment:
I think there's a bunch more code you can blast from orbit because `config-gc` is the only configmap consumed by this controller: https://github.com/knative/serving/blob/52a6e2c08b979b91693ab4503205721f4b30c9be/pkg/reconciler/configuration/config/store.go#L63

If you do it here, then the same logic for the GC controller will show up as moves and be easier to review